sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -1109,10 +1109,10 @@ public partial class PathHierarchyTokenizerV2 : Azure.Search.Documents.Models.To
     public partial class PatternAnalyzer : Azure.Search.Documents.Models.Analyzer
     {
         public PatternAnalyzer(string name) : base (default(string)) { }
-        public Azure.Search.Documents.Models.RegexFlags? Flags { get { throw null; } set { } }
+        public System.Collections.Generic.IList<Azure.Search.Documents.Models.RegexFlags> Flags { get { throw null; } }
```

Comment:
Should the type name be `RegexFlags` if it's being collected in a list?  I feel like a bunch of flags would be represented by `List<RegexFlag>`.  All the other type names in the APIs changed here are singular.

Reply:
That was the model name, but I could change it. It had crossed my mind, but all the other languages still used the model name. I suppose one could argue making this singular is idiomatic, but I was trying not to stray too far from cross-language similarities. That said, I'm open to either.

Reply:
Opened #11510 to track.